demisto_sdk/commands/pre_commit/hooks/docker.py

```diff
@@ -202,27 +218,31 @@ def prepare_hook(
         logger.debug(
             f"Elapsed time to gather tags to files: {end_time - start_time} seconds"
         )
+        if copy_files := self._get_property("copy_files"):
```

Comment:
dont understand what we're doing here.
Copying something from root content path to the directory of each integration? Why?

Reply:
sometimes we need some files to be inside of the directory.
For example, `conftest.py` cannot be provided as an argument and in powershell until we fix PSModulePath we will need to copy demistomock.ps1 and commonserver